CONTRIBUTING.md

```diff
@@ -13,20 +13,19 @@ Conduct][code_of_conduct].
 ## Getting Started
 
 1. To get up and running:
-```
+
+```cmd
 npm install
 npm run dev
 ```
 
-2. To do all code static analysis checks: (DO THIS BEFORE PUSHING COMMITS)
-```
+1. To do all code static analysis checks: (DO THIS BEFORE PUSHING COMMITS)
```

Comment:
should this be a `2`?

Reply:
Syntax highlighting started getting mad at me because apparently Markdown sometimes wants numeric lists with all `1.` labels... Functionally it looks like there's no difference though I can switch this back for consistency.

https://www.markdownguide.org/basic-syntax/#ordered-lists

Reply:
ohhhh. I did not know that! Seems fine either way, I would just notice because I often read the raw markdown file rather having it properly loaded 😆